llvm/lib/AsmParser/LLParser.cpp

```diff
@@ -4274,6 +4274,7 @@ bool LLParser::parseValID(ValID &ID, PerFunctionState *PFS, Type *ExpectedTy) {
   case lltok::kw_bitcast:
   case lltok::kw_addrspacecast:
   case lltok::kw_inttoptr:
+  // ptrtoaddr not supported in constant exprs (yet?).
```

Comment:
That's something that is needed, we support that on CHERI

Reply:
(i.e. it's a TODO if not implemented here, not a question of whether it should be supported, IMO)